freeipa.spec.in

```diff
@@ -592,6 +592,7 @@ Requires: %{name}-common = %{version}-%{release}
 Requires(pre): python3-ldap >= %{python_ldap_version}
 Requires: gnupg2
 Requires: keyutils
+Requires: less
```

Comment:
Please remove the commit with this change set. It's no longer required with the second change.